frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerPanelBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  Title,
+} from '@patternfly/react-core';
+import './GuidedTourDrawer.scss';
+
+type GuidedTourDrawerProps = {
+  expanded: boolean;
+  title: string;
+  tourDuration: number;
+  onClose: () => void;
+  guidedTourContent: JSX.Element;
+  children?: React.ReactNode;
+  additionalHeaderContent?: React.ReactElement;
+};
```

Comment:
The drawer content should receive the following props. The `guidedTour` prop should contain the static information about the tour (ie title, duration, steps, etc). The `guidedTourState` is still TBD but is the information that is tracked and saved in `localStorage`; it includes information such as whether the tour has been completed or is in progress.

Based on this information the drawer content can render all the necessary parts. For now it can render the title. But the contents is beyond the scope of this story.

```suggestion
type GuidedTourDrawerProps = {
  guidedTour: GuidedTour;
  guidedTourState: TBD;
  onClose: () => void;
  children?: React.ReactNode;
  
  expanded: boolean;
};
```

Reply:
done